tools/make_webshots.py

```diff
@@ -38,8 +38,12 @@ def login(driver, username, password):
     password_field.send_keys(password)
     driver.find_elements_by_tag_name("form")[0].submit()
 
-    WebDriverWait(driver, 30).until(
-        EC.presence_of_element_located((By.CLASS_NAME, "v-avatar")))
+    try:
+        WebDriverWait(driver, 30).until(
+            EC.presence_of_element_located((By.CLASS_NAME, "v-avatar")))
```

Comment:
FWIW this is what I see -- so there is indeed v-avatar div
![image](https://user-images.githubusercontent.com/39889/111518510-84210780-872c-11eb-9f99-5548f29f6703.png)